app/views/sections/items/ItemWineGrowingZoneView.scala.html

```diff
@@ -0,0 +1,69 @@
+@*
+ * Copyright 2023 HM Revenue & Customs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *@
+
+@import models.requests.DataRequest
+@import viewmodels.LegendSize
+@import models.sections.items.ItemWineGrowingZone
+@import config.AppConfig
+
+@this(
+        layout: templates.Layout,
+        formHelper: FormWithCSRF,
+        govukErrorSummary: GovukErrorSummary,
+        govukRadios: GovukRadios,
+        continueOrExit: components.continueOrExit,
+        h1: components.h1,
+        h2: components.h2,
+        link: components.link,
+        p: components.p,
+        appConfig: AppConfig
+)
+
+@(form: Form[_], action: Call)(implicit request: DataRequest[_], messages: Messages)
+
+@layout(pageTitle = title(form, messages("itemWineGrowingZone.title")), maybeShowActiveTrader = maybeShowActiveTrader(request)) {
+
+    @formHelper(action, 'autoComplete -> "off") {
+
+        @if(form.errors.nonEmpty) {
+            @govukErrorSummary(ErrorSummaryViewModel(form, errorLinkOverrides = Map("value" -> "value_0")))
+        }
+
+        @h2(messages("items.subHeading", request.draftId), "govuk-caption-xl", hiddenContent = Some(messages("subHeading.hidden")))
+
+        @govukRadios(
+            RadiosViewModel(
+                field = form("value"),
+                legend = LegendViewModel(Text(messages("itemWineGrowingZone.heading"))).asPageHeading(LegendSize.Large),
+                items = ItemWineGrowingZone.options
+            ).withHint(HintViewModel(HtmlContent(hint)))
+        )
+
+        @continueOrExit()
+    }
+}
+
+@hint = {
+    @p() {
+        @messages("itemWineGrowingZone.hint.preLink")
+        @link(appConfig.wineGrowingZoneHintUrl, "itemWineGrowingZone.hint.link", opensInNewTab = true)
```

Comment:
The prototype/copydeck doesn't say it opens in a new tab, although I imagine what you have done is right because we shouldn't interrupt this flow. Maybe worth checking?

Reply:
Should open in New Tab. Takes the User out of the journey.